features/step_definitions/buyer_steps.rb

```diff
@@ -155,6 +155,15 @@ def login_form
   )
 end
 
+Given "a buyer logged in to a provider using SSO" do
+  steps %(
```

Comment:
Reusing steps as if they were function is generally a bad pattern. If possible (it may be too hard to be worth it) replace this steps with their actual implementation.